src/main/webapp/app/assessment/athena.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable, of, switchMap } from 'rxjs';
+import { TextBlockRef } from 'app/entities/text-block-ref.model';
+import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
+
+@Injectable({ providedIn: 'root' })
+export class AthenaService {
+    public resourceUrl = 'api/athena';
+
+    constructor(protected http: HttpClient, private profileService: ProfileService) {}
```

Comment:
codacy issue, see also the changes in #7004.

Reply:
I tried doing that, but the ESLint pre-commit hook always seems to automatically undo my change. I'll try changing it in the next commit again.